projects/core/src/model/cart.model.ts

```diff
@@ -82,6 +82,10 @@ export interface SaveCartResult {
   savedCartData?: Cart;
 }
 
+export interface CartList {
```

Comment:
I see it only used in one spec file. In this case it should not be added

Reply:
Can't we keep it?

Future cart things are coming soon such as import saved cart(s) and export of cart(s)

I believe it's good to always have a defined interface to be used anyhow

Reply:
removed